llms/mlx_lm/utils.py

```diff
@@ -184,8 +238,8 @@ def generate(
         if token_count == 0:
             print("No tokens generated for this prompt")
             return
-        prompt_tps = prompt.size / prompt_time
-        gen_tps = (token_count - 1) / gen_time
+        prompt_tps = prompt_tokens.size / prompt_time
+        gen_tps = (len(token_count) - 1) / gen_time
```

Comment:
Nit: `token_count` is already a number, there's no need to apply `len(token_count)` again.